KickoffTimerPlugin.cpp

```diff
@@ -1,28 +1,46 @@
 #include "KickoffTimerPlugin.h"
-#include "bakkesmod\wrappers\includes.h"
+#include "bakkesmod/wrappers/includes.h"
 #include "utils/parser.h"
 #include <string>
+#include <stdio.h>
+#include <filesystem>
 
 BAKKESMOD_PLUGIN(KickoffTimerPlugin, "Kickoff timer plugin", "0.1", PLUGINTYPE_FREEPLAY)
 
-static const string savefile = "bakkesmod/data/kickofftimerplugin.data";
+string savefile = "bakkesmod/data/kickofftimerplugin.data";
+enum event_types { normal, better_than_normal, new_pb, slow_time };
+map<event_types, Color> color_mapping =
+{
+	{ normal,             { 255, 255, 255 }}, // White
+	{ better_than_normal, { 255, 200, 0 }}, // Yellow-ish
+	{ new_pb,             { 0, 255, 0 }}, // Green
+	{ slow_time,          { 255, 0, 0 }} // Red
+};
+
+int DECIMAL_PRECISION;
 
 void KickoffTimerPlugin::onLoad()
 {
 	spawnLocations.push_back({ "ML", {256, -3840}, 2.30312 });
 	spawnLocations.push_back({ "MR", {-256, -3840}, 2.30312  });
-	spawnLocations.push_back({ "M", {0, -4608}, 2.62897 });
-	spawnLocations.push_back({ "L", {2048, -2560}, 2.06905 });
-	spawnLocations.push_back({ "R", {-2048, -2560},  2.06905 });
+	spawnLocations.push_back({ "M",  {0, -4608}, 2.62897 });

```

Comment:
Wrong Spaces

Reply:
I'm not sure what you mean here. Could you explain it a little more?

Reply:
There are 2 spaces between "," and "{"

Reply:
Its just a formatting thing to make it align in the IDE. It doesn't affect functionality, but I can remove it